eng/Versions.props

```diff
@@ -328,8 +328,8 @@
     <XunitExtensibilityExecutionVersion>$(XunitVersion)</XunitExtensibilityExecutionVersion>
     <XUnitRunnerVisualStudioVersion>2.8.2</XUnitRunnerVisualStudioVersion>
     <MicrosoftDataSqlClientVersion>5.2.2</MicrosoftDataSqlClientVersion>
-    <MicrosoftOpenApiVersion>1.6.17</MicrosoftOpenApiVersion>
-    <MicrosoftOpenApiReadersVersion>1.6.17</MicrosoftOpenApiReadersVersion>
+    <MicrosoftOpenApiVersion>2.0.0-preview2</MicrosoftOpenApiVersion>
+    <MicrosoftOpenApiReadersVersion>2.0.0-preview2</MicrosoftOpenApiReadersVersion>
```

Comment:
Are we planning to upgrade to 2.0.0 RTM once it's available? The build will eventually complain if we're still on `preview2` when we stabilize for 10.0

Reply:
Yep! There will eventually be an RTM of the package but we're doing this work now to provide feedback to the Microsoft.OpenApi team about the APIs and iterate on things.

> The build will eventually complain if we're still on preview2 when we stabilize for 10.0

When do we expect this to happen?

Reply:
> When do we expect this to happen?

The second half of next year some time, so we've got plenty of time